com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -47,21 +47,21 @@ public static function getContactRoles($cuid)  {
   }
 
   /**
-   * Get options for a given job roles field along with their database IDs.
+   * Get option values for specific option group.
    *
    * @param String $fieldName
    *
-   * @return Array
+   * @return array
    */
   public static function buildDbOptions($fieldName) {
     $queryParam = array(1 => array($fieldName, 'String'));
-    $query = "SELECT cpv.id, cpv.label from civicrm_option_value cpv
+    $query = "SELECT cpv.value, cpv.label from civicrm_option_value cpv
               LEFT JOIN civicrm_option_group cpg on cpv.option_group_id = cpg.id
               WHERE cpg.name = %1";
     $options = array();
     $result = CRM_Core_DAO::executeQuery($query, $queryParam);
     while ($result->fetch()) {
-      $options[] =  array( 'id'=>$result->id, 'label'=>strtolower($result->label) );
+      $options[] =  array( 'value'=>$result->value, 'label'=>strtolower($result->label) );
     }
     return $options;
   }
```

Comment:
if I want to use buildOptions method it will require me to add sample option values in the tests via this method  CRM_Core_BAO_OptionValue::add due to way civicrm cache option values so inserting option values via sql query will never work probably with buildOptions ... but unfortunately using CRM_Core_BAO_OptionValue::add to add option values was always cause civix to hit the time limit so I dropped using it and stayed with my method.